tools/clang/test/CodeGenSPIRV/method.structured-buffer.load.hlsl

```diff
@@ -0,0 +1,39 @@
+// Run: %dxc -T vs_6_0 -E main
```

Comment:
according to https://msdn.microsoft.com/en-us/library/windows/desktop/ff471494(v=vs.85).aspx
RWStructuredBuffer is only supported for Pixel Shader and Compute Shader